bindata/v4.1.0/kube-apiserver/defaultconfig.yaml

```diff
@@ -106,6 +106,11 @@ servingInfo:
   requestTimeoutSeconds: 3600
 serviceAccountPublicKeyFiles:
   - /etc/kubernetes/static-pod-resources/configmaps/sa-token-signing-certs
+  - /etc/kubernetes/static-pod-certs/configmaps/bound-sa-token-signing-certs
+serviceAccountIssuer: auth.openshift.io
```

Comment:
please pass as a flag, not as a struct value.  Same with all these values.

Reply:
You're looking for `apiServerArguments` above and add comments to help future me who won't remember this as well.

Reply:
Done